executor/select_into.go

```diff
@@ -121,26 +121,31 @@ func (s *SelectIntoExec) dumpToOutfile() error {
 				(encloseFlag && encloseOpt && s.considerEncloseOpt(et)) {
 				s.lineBuf = append(s.lineBuf, encloseByte)
 			}
-			switch col.GetType().EvalType() {
-			case types.ETInt:
+			switch col.GetType().Tp {
+			case mysql.TypeTiny, mysql.TypeShort, mysql.TypeInt24, mysql.TypeLong:
```

Comment:
Why this branch doesn't need to consider signed or unsigned?

Reply:
For these types, we ignore their signs and store them simply as `int64` in TiDB since there is no risk to overflow.
I add one more test for this.
PTAL @winoros

Reply:
Can we load this file into mysql successfully if we ignore the signs?

Reply:
Sure, I test it successfully in local:
```
mysql> create table t (v tinyint unsigned);
Query OK, 0 rows affected (0.01 sec)

mysql> load data local infile '/tmp/rrrr' into table t;
Query OK, 1 row affected (0.00 sec)
Records: 1  Deleted: 0  Skipped: 0  Warnings: 0

mysql> select * from t;
+------+
| v    |
+------+
|  127 |
+------+
1 row in set (0.00 sec)
```